lib/segment/src/id_tracker/compressed/versions_store.rs

```diff
@@ -0,0 +1,74 @@
+use std::collections::HashMap;
+
+use crate::types::SeqNumberType;
+
+/// Compressed representation of Vec<SeqNumberType>
+/// Which takes advantage of the fact that the sequence numbers are likely to be < 2**32
+///
+/// Implements a required subset of the Vec API:
+///
+/// * get by index
+/// * set by index
+/// * [] operator
+/// * len
+/// * push
+#[derive(Debug)]
+pub struct CompressedVersionsStore {
```

Comment:
`Store` sounds like it is persisted, maybe just omit it? 
```suggestion
pub struct CompressedVersions {
```